pkg/nvpci/nvpci.go

```diff
@@ -521,6 +528,31 @@ func getDriver(devicePath string) (string, error) {
 	return "", err
 }
 
+// /sys/bus/pci/devices/0000:df:00.0/vfio-dev/vfio0.
+func getIOMMUFD(devicePath string) (int, error) {
+	vfioDevDir := filepath.Join(devicePath, "vfio-dev")
+	// Read the directory; expect exactly one entry
+	entries, err := os.ReadDir(vfioDevDir)
+	switch {
+	case os.IsNotExist(err):
+		return -1, nil
+	case err == nil:
+		if len(entries) == 0 {
+			return -1, fmt.Errorf("no VFIO device file found in %s", vfioDevDir)
+		}
+		name := entries[0].Name()
+		// Strip the "vfio" prefix to get the numeric part
+		idxStr := strings.TrimPrefix(name, "vfio")
+		idx, err := strconv.Atoi(idxStr)
```

Comment:
Nit: Should we use `ParseInt` here and specify a 64 bit width as we do for the IOMMU Group?